.github/workflows/tests-cli.yml

```diff
@@ -89,14 +89,15 @@ jobs:
           python -m pytest tests/cli/
 
   push-to-tinybird:
-    if: always() && github.ref == 'refs/heads/master'
+    # if: always() && github.ref == 'refs/heads/master'
+    if: always()
     runs-on: ubuntu-latest
     needs: cli-tests
     steps:
       - name: Push to Tinybird
-        uses: localstack/tinybird-workflow-push@v3.0.0
+        uses: localstack/tinybird-workflow-push@v3
         with:
           workflow_id: "tests_cli"
           tinybird_token: ${{ secrets.TINYBIRD_CI_TOKEN }}
           github_token: ${{ secrets.GITHUB_TOKEN }}
-          tinybird_datasource: "ci_workflows"
+          tinybird_datasource: "ci_workflows_test"
```

Comment:
Looks like you are using a test datasource here still?

Reply:
Just added this quickly right now to test out if the major version tag really works. Will remove shortly